lib/puma/launcher.rb

```diff
@@ -195,7 +198,7 @@ def connected_port
     def restart_args
       cmd = @options[:restart_cmd]
       if cmd
-        cmd.split(' ') + @original_argv
+        cmd.split + @original_argv
```

Comment:
I don't think we should remove this, as it depends on `$;` being nil.

Reply:
I've seen that a lot of places, including Ruby core, but I'll change it.